src/main/java/com/freeit/anaron/lesson3/Task1.java

```diff
@@ -0,0 +1,17 @@
+package com.freeit.anaron.lesson3;
+
+//1)Начав тренировки, спортсмен в первый день пробежал 10 км.
+//Каждый день он увеличивал дневную норму на 10% нормы предыдущего дня.
+//Какой суммарный путь пробежит спортсмен за 7 дней?
+public class Task1 {
+    public int findDistance(int allDay) {
+        double kilometer = 10;
+        int countDay = 1;
```

Comment:
please read the task more carefully.
Fist day 10km, second day 10+10%=11km. So two days distance is 21km.
Your program gives incorrect result